docker-compose.test.parallel.yaml

```diff
@@ -27,8 +27,9 @@ services:
+      API_PRIVATE_KEY: "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV2Z0lCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktnd2dnU2tBZ0VBQW9JQkFRRFdKTCs3TExDMmx0SlAKTkdvb0FzUTE2Y0l2R2hzU0lub2liRHQ2TGhtdFdqRWtYb1M3UUxOS0VhYzIyRXdwSjRDcTZ1cHRRSmtRSCsyVQpnNmltNko2aUVwR0hMZENoQVJlMFNhNVlWcUdiZzd1cDkrbGlXMkNiQkZsV3FVeEg4ZEorZUlIL3BCVEJ5Y3VtCkdDcHdvd1FUZlFCZDNrUmVXOEllU3lGZWJUMm9BVUw4YTZEMFBsTTJydThEb25ZeXY4QzJtcU5QVzNNU3NINVIKd1J1WlhBZWlxa1Y4RWxkOGc3OStJY0pXM0g1L081a2J3Z1RuUDhFYkI2MldHMklBa2twQWdnWXdxem8vUjhqUAo2dzY4Q0JNK29oZ3UvU1Aza2xxM2hUanRTbFFxM3N2ZlRVTlVaNmlLQ0hOQkhYRmlyUGRGUlVYT0xIMGpvb2xVCjZnZjEzSDM5QWdNQkFBRUNnZ0VBQXlsZWthS2hYRWRzbWt5WU9hTFZmSm56Z0QyeGd4L0hyNnpyUERkUE1UMlkKNXFRazc3SjVtSUViWW51b2lqTlhXcmxSQUkvRGdzR3ZRTEg5N04yWDJWQTVYbEVqN2Z0RnV6ekZsUnA0VklNSwpJNjBHNjM1cVlNemNZeU5vNlNkWWw5TkFlMjFUMGlCSzFheGJLZzBQYThNOHgyWUI3K1plYVdpV0Rra1hiOXpICitTMmZneEtpWWZQclV2MXB3WnhLL3JZV1pCL2Q5L0xTQUtveWQrQmNqQ1B2RnYwbXZiY0k4ZG1UTE5hYU1RNHoKUlBqV2tvOWpWRmhGYUdCOWZMVEFNVE9Db0NTQzlrLzFWQkVwMm5WUjBPQnAzbU9scDJGVDN4KzVza2RNbi9yQgpZUWRJelo3Z1RKaGxXbFVpRGsxWXB2M0tPUVlYaW40YTNYTHhuVjNXRVFLQmdRRDhOeUNqWW9SWXpSeTQwUTZ3Cm1TVmlkSWM5SWpQV0VUam1ia1R5MjNJL0kxVWpHUkxqNktjNzBVaTJVSFpLWGpaQktGQnhyb1Y0ckxnSW13NGYKeGNnbDFiWUZNWW14RFpFZmdUOElmMGJZelZ4Vmh5WXgrS1VTOHdkWmtOQWd6VjhhZmJMKysvRzdqd0N6eisyNQpqSk5rWlBTTVJ1b0gxMlpQWWc2d05lNmlqUUtCZ1FEWlcxN2ZJcHdTSFBDL0k5bzlETlhOekZFOUo2OUZ2b1RiCmJQMXZrQk9FclZScGQ5QkdJTGd6K3RsQ3hYQTZRcjBwRTBaT0ZYQnQyZHRpWm9UY2Zma25VWFJHd3FPUjF1c3EKSmtLUlNNVHlzTFhQVWJ6Zmd5ZHJyTEsvUjhEWW01aURLTm9OYlVydVlJM1Y5WHA5bEpsVkRXV1hlQ082WldYSAowRXE4R0xVbE1RS0JnUUN4ZDVETnBpcDkyWVdXUGZQYUswaDA2STJvWklLWTQvRm5vcnQ5THhCS2dQQU5ERDJRCmhFbjMreUR3dDlra0p2cUNDbjNnSCs3bXBKR3g3SGladnE5QWdBbmR5YWViSDNLMytsZUNBUnkydkhsMTdPemgKU0pYT3FaYklYTzgxeFl4ZnRZYlhpQ3ZsejQrczltZk5vYTZzZTZUZmQ1NkErZjJSYWV2djdXTUdtUUtCZ1FETwpVQk5pVzVpbS9MOXJ2dDFGTzZ5cDFubDZRZlAxa0p6TkpzdjlIRlhhaHlmcm9HV2FXeTVkY0lyTy9WZkVoVnJGCjVWekRlMGdualRYWFZoWmdzRlpTQXRnSzNtK2lXODBHU2lDL2hNSWsxcVVNYjdxR3NoZzE5VXdKa2xNemg2V2gKUWFLUVFpQncya2RuUXNZTTg0TURuMTdDQXJCdkdoWWRMdEh4YnNacTBRS0JnSFZIUlJKY2RKSVVsSU9uaWZhNQpYS1FyZUNLa3VkWEtLY2dmaGt5UllrVVIrcFp1ZFA4SXZSSEFiVmRQQjU1emFNUHZyUlBmY1VGcnQyQk9RMDdNCmVnUHNLVnVremdkVUVSVU5MUEszYzEyK1FLQ0IwaTRkaWFPRUVzMDRoTGRNWVFYdW9waUZwT0VxMlRiRk1BRHgKRHVFbUpzaUxjSkZVOEVYMytLOXlxRnVhCi0tLS0tRU5EIFBSSVZBVEUgS0VZLS0tLS0K"
+      API_PUBLIC_KEY: "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUExaVMvdXl5d3RwYlNUelJxS0FMRQpOZW5DTHhvYkVpSjZJbXc3ZWk0WnJWb3hKRjZFdTBDelNoR25OdGhNS1NlQXF1cnFiVUNaRUIvdGxJT29wdWllCm9oS1JoeTNRb1FFWHRFbXVXRmFobTRPN3FmZnBZbHRnbXdSWlZxbE1SL0hTZm5pQi82UVV3Y25McGhncWNLTUUKRTMwQVhkNUVYbHZDSGtzaFhtMDlxQUZDL0d1ZzlENVROcTd2QTZKMk1yL0F0cHFqVDF0ekVyQitVY0VibVZ3SApvcXBGZkJKWGZJTy9maUhDVnR4K2Z6dVpHOElFNXovQkd3ZXRsaHRpQUpKS1FJSUdNS3M2UDBmSXorc092QWdUClBxSVlMdjBqOTVKYXQ0VTQ3VXBVS3Q3TDMwMURWR2VvaWdoelFSMXhZcXozUlVWRnppeDlJNktKVk9vSDlkeDkKL1FJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg=="
+      ENFORCEMENT_URL: https://enforcement.com
```

Comment:
I think you need to add back in AUTH_SERVER_URL and USER_FEEDBACK_EMAIL ADDRESS vars